sdk/tests/conformance2/context/methods-2.html

```diff
@@ -188,6 +188,7 @@
 "invalidateSubFramebuffer",
 "readBuffer",
 "renderbufferStorageMultisample",
+"texImage3D",
 "texStorage2D",
 "texStorage3D",
 "texSubImage3D",
```

Comment:
I noticed we have multiple texSubImage3D (five of them).  Is this necessary?  If not, maybe we should clean it up in this CL.